scorecards-site/modules/Footer/Footer.vue

```diff
@@ -12,7 +12,9 @@
           ]"
       /></a>
       <div class="text-14 mb-12">
-        © 2022 The Linux Foundation, under the terms of the Apache License 2.0.
+        Copyright © OpenSSF Scorecard a Series of LF Projects, LLC
+        <br />
+        For web site terms of use, trademark policy and other project policies please see https://lfprojects.org.
```

Comment:
```suggestion
        For website terms of use, trademark policy and other project policies please see https://lfprojects.org.
```

I think this should be one word, but a quick check with LF Legal would be good.

Reply:
@justaugustus Message sent to LF Legal on whether we can change to "website" and add missing commas.

Reply:
Can we also make the link a link?